src/bootstrap/config.rs

```diff
@@ -1212,6 +1222,16 @@ impl Config {
         let mut omit_git_hash = None;
 
         if let Some(rust) = toml.rust {
+            config.download_rustc_commit = config.download_ci_rustc_commit(rust.download_rustc);
+            // This list is incomplete, please help by expanding it!
```

Comment:
I'm not sure what list this is referring to, say more?

Reply:
this is referring to the `check_ci_rustc` checks (and indirectly to https://rust-lang.zulipchat.com/#narrow/stream/326414-t-infra.2Fbootstrap/topic/ban.20rustc.20configs.20when.20download-rustc.20is.20enabled.3F and the checks we already do for `download-ci-llvm`). right now this only bans the options that I've manually tested to break; it doesn't ban options that are ignored or that I haven't manually tested. the idea is that if other people run into breakage because of an option set locally, they'd add it to the banned options here.

Reply:
Thanks, sounds good.